### PR TITLE
(feature): added ci/cd docker actions #6

### PR DESCRIPTION
**Pull request by Rashid-007**
Completion of CI/CD

### CODE REVIEW

File: .github/workflows/skillmatrix-tool.yml [hunk 1]
```diff
@@ -1,17 +1,88 @@
...
+    branches:
+      - development
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
+    env:
+      DOCKER_USERNAME: ${{ secrets.BBV_DOCKER_USERNAME }}
```
Comment (joschkawanke) [inline .github/workflows/skillmatrix-tool.yml]: Remove wenn es nicht benötigt wird

File: .github/workflows/skillmatrix-tool.yml [hunk 2]
```diff
@@ -1,17 +1,88 @@
...
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
       - name: Build with Gradle
-        run: ./gradlew build
+        run: ./gradlew test
+
+
+  container-build:
+    #if: github.ref == 'refs/heads/development'
```
Comment (joschkawanke) [inline .github/workflows/skillmatrix-tool.yml]: ?? Remove it ore comment it

File: .github/workflows/skillmatrix-tool.yml [hunk 3]
```diff
@@ -1,17 +1,88 @@
...
+  dev-deployment:
+    #if: github.ref == 'refs/heads/development'
+    needs: [build, container-build]
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_USERNAME: ${{ secrets.BBV_DOCKER_USERNAME }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Package the deployment artifact
+        run: zip -r skillmatrix-tool.zip Dockerrun.aws.json
```
Comment (joschkawanke) [inline .github/workflows/skillmatrix-tool.yml]: Please Rename skillmatrix-tool into DeploymentReferenzForSkillmatrixDocker

File: .github/workflows/skillmatrix-tool.yml (hunk 1 above)
Comment (Rashid-007) [reply .github/workflows/skillmatrix-tool.yml]: Ok

File: .github/workflows/skillmatrix-tool.yml (hunk 3 above)
Comment (Rashid-007) [reply .github/workflows/skillmatrix-tool.yml]: renamed

File: .github/workflows/skillmatrix-tool.yml (hunk 2 above)
Comment (Rashid-007) [reply .github/workflows/skillmatrix-tool.yml]: Removed

**Pull request merged**